parlai/core/torch_agent.py

```diff
@@ -1295,41 +1295,52 @@ def observe(self, observation):
                               text_truncate=self.text_truncate,
                               label_truncate=self.label_truncate)
 
+    def get_save_dict(self):
```

Comment:
can we rename this something that makes it clear this is referring to the state dict and not hte dictionary? maybe like `get_state_dict` or `get_save_state_dict`?